### DISCUSSION TITLE
Niagara Falls is located between which two Great Lakes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#379] [Galaxy Trivia] [answered]
**Great Lakes:**  HOMES

**Comment by Roberto-Larivera** [comment]
Le Cascate del Niagara si trovano tra il Lago Ontario a nord e il Lago Erie a sud. Il fiume Niagara che alimenta le cascate scorre dal Lago Erie al Lago Ontario.

**Comment by peterbaccaro** [comment]
The Niagara Falls connects the **Erie** and **Ontario** Great Lakes.

**Comment by thuongtruong1009** [comment, marked as answer]
Niagara Falls is not located between two Great Lakes. Rather, it is located between Lake Erie and Lake Ontario. The Niagara River flows from Lake Erie to Lake Ontario and forms the border between the province of Ontario in Canada and the state of New York in the United States.
